### ISSUE TITLE
Newline with 'o' key causes text highlighting

### ISSUE THREAD

**Issue by jbloggz** [VSCodeVim/Vim#3684]
**Describe the bug**
When using the 'o' key to start a new line, all new text is highlighted red until you change to a new tab then back

**To Reproduce**

1.  Go to the middle of a file
2.  Press the 'o' key to start a new line
3.  Start typing

**Expected behavior**
The text shouldn't be highlighted

**Screenshots**
![image](https://user-images.githubusercontent.com/29333413/56253288-106cbf00-60ff-11e9-9e51-6b886abbac05.png)

**Environment (please complete the following information):**

- Extension (VsCodeVim) version: 1.4.0
- VSCode version: 1.33.0
- OS: Ubuntu

**Comment by jbloggz** [comment]
This issue seems to be related to the 'Trailing Spaces' extension. It only happens when this extension is enabled

**Comment by J-Fields** [comment]
Are you still seeing this @jbloggz? I use the trailing spaces extension and have not experienced this.

**Comment by jbloggz** [comment]
Yes, I just reinstalled the trailing spaces extension and I still get the issue.
If i start a new line by pressing 'enter' from 'insert' mode, then it's all fine. But if I start a new line by pressing 'o' from 'normal' mode, then all my new text will stay highlighted indefinitely (as can be seen in my screenshot above)

**Comment by jbloggz** [comment]
Here is an example of it happening. First I use 'enter' in 'insert' mode, which works, then 'o' in 'normal' mode, which breaks:
![Peek 2020-04-30 23-49](https://user-images.githubusercontent.com/29333413/80718480-89d62480-8b3d-11ea-8428-fa95f41560c6.gif)
